dspace/config/ldn/request-endorsement-resubmission

```diff
@@ -0,0 +1,56 @@
+## generate LDN message json when request-endorsement of an item
+##
+## Parameters: {0} config 'dspace.ui.url' OR supplied actor=id (mailto:sample@email.org)
+##             {1} config 'ldn.notify.inbox'
+##             {2} config 'dspace.name'
+##             {3} Notify Service url
+##             {4} Notify Service ldnUrl
+##             {5} 'dspace.ui.url'/handle/xxxx/yyy
+##             {6} metadata value of 'dc.identifier.uri'
+##             {7} the url to the primary bitstream or the first bitstream in the ORIGINAL bundle if there is no primary bitstream. The url is 'dspace.ui.url'/bitstreams/:uuid/download
+##             {8} the bitstream  MimeType or get User Format MimeType if getFormat is 'Unknown'
+##             {9} id of the created LDNMessage
+##             {13} type of actor: Service or Person
+##             {14} config 'dspace.ui.url'
```

Comment:
same as for the other file (referring to parameter number 14)

> Is there a reason why a new parameter was added even though from the description above I see that the parameter is equal to parameter 0? I think it's probably a typo in the description of this file and the parameter is not always intended to be equal to parameter 0. In this case a better description of the parameter might be needed

Reply:
Kept as needed and amended description